### ISSUE TITLE
[Accessibility Audit] IAM and Admin Exp: Sign in

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34112]
### Steps to replicate journey

Taken from the [user journey list document](https://docs.google.com/document/d/1h3bh6prxPX31aOiWkNsRCvXUbmtakaaj3Kn986uZYkc/edit?usp=sharing). Use this for further context.

With email:

- Open the main sourcegraph page (sourcegraph.com)
- Click on the “Log in” button on the top right
- You should see the login form
- Fill in your username, password and click login

With GitHub / GitLab:

- Same as above, but instead of signing in with email, click on the relevant button
- Follow the prompts, if any (out of our control)
  - We do not need to audit external sites that we cannot control.

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

### Any other questions?

See the [audit FAQ](https://docs.google.com/document/d/1nCi7TvYRpj9eG0_QtgqK1c4L28EiJKVuxyuvg1lAGgo/edit#) - add your question if it isn't there already!

**Comment by sourcegraph-bot-2** [comment]
Heads up @sourcegraph/iam-and-admin-experience - the "team/iam-admin-experience" label was applied to this issue.

**Comment by sourcegraph-bot-2** [comment]
Heads up @sourcegraph/iam - the "team/iam" label was applied to this issue.
